x-pack/plugins/apm/public/components/shared/charts/Timeline/TimelineAxis.js

```diff
@@ -74,7 +74,7 @@ function TimelineAxis({ plotValues, agentMarks, traceRootDuration }) {
                 }}
               />
 
-              {traceRootDuration && (
+              {traceRootDuration > 0 && (
```

Comment:
This is a good catch. My only (small) concern with this fix is that I am now wondering "oh can duration be negative and we only want to render if it's positive?" which isn't true, we're just trying to force this to be boolean. I have a slight preference for switching these to ternaries and/or using `!!traceRootDuration`, but perhaps that's overkill.

Reply:
> My only (small) concern with this fix is that I am now wondering "oh can duration be negative and we only want to render if it's positive?" which isn't true,

We want to make sure, that the number is larger than zero. In my view `!!traceRootDuration` would hide this fact even more.

Reply:
I can also change it to use a ternary if you prefer:

```js
              {traceRootDuration > 0 ? (
                <LastTickValue
                  x={xScale(traceRootDuration)}
                  value={tickFormat(traceRootDuration)}
                  marginTop={28}
                /> : null
              )}
```